npc/re/instances/MalangdoCulvert.txt

```diff
@@ -784,9 +784,8 @@ OnEnable:
 	areamonster(.@map$, .@c[0], .@c[1], .@c[2], .@c[3], _("Abysmal Cornutus"), MD_CORNUTUS, rand(1, 3), .@label$);
 	specialeffect(EF_MAPPILLAR2, ALL_SAMEMAP); //currently broken
 	getmapxy(.@map$, .@x, .@y, UNITTYPE_NPC);
-	getpartymember('party_id, 2);
-	copyarray(.@partymemberaid[0], $@partymemberaid[0], $@partymembercount);
-	for(.@i = 0; .@i<$@partymembercount; ++.@i) {
+	.@partymembercount = getpartymember(getcharid(CHAR_ID_PARTY), PT_MEMBER_ACCID, .@partymemberaid);
```

Comment:
```suggestion
	.@partymembercount = getpartymember('party_id, PT_MEMBER_ACCID, .@partymemberaid);
```

This script runs without a RID attached, it needs to retrieve the party id from the instance variable